packages/react-components/src/organisms/MembersSection.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import css from '@emotion/css';
+
+import { Link, Card, Headline2, Avatar, Paragraph } from '../atoms';
+import { perRem, mobileScreen } from '../pixels';
+
+type MembersSectionProps = {
+  readonly members: {
+    readonly id: string;
+    readonly firstName?: string;
+    readonly lastName?: string;
+    readonly displayName: string;
+    readonly role: string;
+    readonly avatarURL?: string;
+  }[];
+};
+
+const containerStyles = css({
+  padding: 0,
+  listStyle: 'none',
+
+  display: 'flex',
+  flexDirection: 'row',
+  flexWrap: 'wrap',
+});
+
+const avatarContainer = css({
+  flexBasis: 1,
+});
+
+const detailsContainer = css({
+  paddingLeft: `${12 / perRem}em`,
+});
+
+const memberContainerStyles = css({
```

Comment:
We can merge this / not block the PR on this for now, but the layouting needs a bit of work, it has weird states atm
![image](https://user-images.githubusercontent.com/16069751/90806891-f445e700-e31d-11ea-857d-199520f043e6.png)
The media query should probably be removed in favor of automatic wrapping since the text sizes are unknown, and this should probably be an `auto-fill` grid or so, not a flex, since otherwise there can be holes
![image](https://user-images.githubusercontent.com/16069751/90807539-e9d81d00-e31e-11ea-81ec-2e3985352bd5.png)
Let me know if you want me to take over part of this

Reply:
We can discuss this when you're back :)

Reply:
I might also be able to fit something in tomorrow after finishing the current ticket